automation/src/test/scala/org/broadinstitute/dsde/firecloud/test/billing/BillingSpec.scala

```diff
@@ -23,110 +23,115 @@ class BillingSpec extends FreeSpec with WebBrowserSpec with UserFixtures with Cl
               api.workspaces.waitForBucketReadAccess(billingProjectName, workspaceName)
               api.importMetaData(billingProjectName, workspaceName, "entities", TestData.SingleParticipant.participantEntity)
 
               val methodConfigName: String = "test_method" + UUID.randomUUID().toString
               api.methodConfigurations.copyMethodConfigFromMethodRepo(billingProjectName, workspaceName, SimpleMethodConfig.configNamespace,
                 SimpleMethodConfig.configName, SimpleMethodConfig.snapshotId, SimpleMethodConfig.configNamespace, methodConfigName)
 
-              withWebDriver { implicit driver =>
-                withSignIn(user) { _ =>
-                  // verify running a method
-                  val methodConfigDetailsPage = new WorkspaceMethodConfigDetailsPage(billingProjectName, workspaceName, SimpleMethodConfig.configNamespace, methodConfigName).open
-                  val submissionDetailsPage = methodConfigDetailsPage.launchAnalysis(MethodData.SimpleMethod.rootEntityType, TestData.SingleParticipant.entityId)
+              // verify running a method
+              val methodConfigDetailsPage = new WorkspaceMethodConfigDetailsPage(billingProjectName, workspaceName, SimpleMethodConfig.configNamespace, methodConfigName).open
+              val submissionDetailsPage = methodConfigDetailsPage.launchAnalysis(MethodData.SimpleMethod.rootEntityType, TestData.SingleParticipant.entityId)
+
+              submissionDetailsPage.waitUntilSubmissionCompletes()
+              assert(submissionDetailsPage.verifyWorkflowSucceeded())
+              //END: Test running anlysis in workspace
+
+              //BEGIN: Test adding user to project
+              billingPage.open.openBillingProject(billingProjectName)
+              billingPage.addUserToBillingProject(secondUser, "User")
+
+              val isSuccess = billingPage.isUserInBillingProject(secondUser)
+              isSuccess shouldEqual true
+              //END: Test adding user to project
 
-                  submissionDetailsPage.waitUntilSubmissionCompletes()
-                  assert(submissionDetailsPage.verifyWorkflowSucceeded())
-                }
-              }
+              //BEGIN: should be able to change the billing account associated with the project
+              val originalBillingAccount = Google.billing.getBillingProjectAccount(billingProjectName)
+              originalBillingAccount shouldBe Some(Config.Projects.billingAccountId)
+
+              Google.billing.removeBillingProjectAccount(billingProjectName)
+
+              val newBillingAccount = Google.billing.getBillingProjectAccount(billingProjectName)
+              newBillingAccount shouldBe None
+              //END: should be able to change the billing account associated with the project
             }
           }
         }
       }
     }
   }
+
+  private def createNewBillingProject(user: Credentials, billingPage: BillingManagementPage, trials: Int = 3): String = {
```

Comment:
This bothers me for 2 reasons:
1. We will not notice these log messages and will therefore not be aware of changes in reliability. Getting logs into Kibana could help with this, especially if we can set up an alert if it happens too frequently.
2. With 3 retries, we are saying that we accept as low as ~33% reliability. A secondary check that gives us an alert while not failing the test could help with this.